documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -4,3 +4,6 @@
 
 ### New features
 * Support for GoLang is now extended to Go 1.22.2.
+### Resolved issues
+* Resolved an issue that could prevent controlling the image inspector HTTP request service timeout through the property *service.timeout*.
+* (IDETECT-3181) Resolved an issue with naming of Eclipse components to improve matching. 
```

Comment:
@sig-cpotts

Reply:
@shantyk guessing that the top item was an internal discovery, hence no Jira number?
Potential tweak for item 2? * (IDETECT-3181) Improved Eclipse component matching implementation through better handling of external identifiers.

Reply:
Line 8 was entirely accidental (Its from an old release, I was using it to make sue I had the right syntax for a "### Resolved issues" section). Thank you so much for pointing that out, removed!

Reply:
+ wording fixed for 3181 note